CONTRIBUTING.md

```diff
@@ -8,6 +8,21 @@ We gratefully welcome improvements to documentation as well as to code.
 
 ## Getting Started
 
+### Pre-requisites
+
+* [Go](https://golang.org/doc/install).
+* Docker version 23.0.0 or greater.
+
+### Local development cheat sheet
+
+* `make test` to run unit tests
+* `make lint` to run linters
+* `make fmt` to format Go code
+* `make vet` to run `go vet`
+* `make generate` to generate code and manifests based on Go struct definitions for CRDs.
+
+`make precommit` includes all of the above.
+
 ### Workflow
 
 It is recommended to follow the ["GitHub Workflow"](https://guides.github.com/introduction/flow/). When using [GitHub's CLI](https://github.com/cli/cli), here's how it typically looks like:
```

Comment:
should we include running precommit as part of the workflow?

Reply:
Done.